### PR TITLE
ci/npm publish

### PR DESCRIPTION
**Pull request by erik-beus**
Allow publishing to npm from Github Actions

### CODE REVIEW

File: packages/ssr/package.json [hunk 1]
```diff
@@ -1,9 +1,8 @@
 {
   "name": "@toddledev/ssr",
-  "version": "0.0.2-alpha.6",
+  "version": "0.0.2-alpha.13",
```
Comment (erik-beus) [inline packages/ssr/package.json:3]: This was already published to npm

File: bunfig.toml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+[install]
+saveTextLockfile = true
```
Comment (jacob-kofoed) [inline bunfig.toml:2]: Not sure what the consequences would be if someone is on bun 1.1.38 or older 🤔

**Review by jacob-kofoed — APPROVED** [review]
I didn't test publish workflow, but looks correct. Do we need to write in README how publishing works? Where to bump versions etc.

**Comment by erik-beus** [comment]
> I didn't test publish workflow, but looks correct. Do we need to write in README how publishing works? Where to bump versions etc.

Good point. I'll get that added (in a followup PR)

**Pull request merged**